toptal/davinci-github-actions
# chore: update set-output usage

[FX-3708]

### Description

`set-output` and `set-state` are deprecated https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files

### How to test

- check [Draft PR in Picasso](https://github.com/toptal/picasso/pull/3377), there are no deprecation warnings anymore

### Development checks


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>


[FX-3708]: https://toptal-core.atlassian.net/browse/FX-3708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
